WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderDetailsDataSource.swift

```diff
@@ -217,68 +180,30 @@ private extension OrderDetailsDataSource {
         }
     }
 
-    func configureBillingAddress(cell: CustomerInfoTableViewCell) {
-        let billingAddress = order.billingAddress
-
-        cell.title = NSLocalizedString("Billing details", comment: "Billing title for customer info cell")
-        cell.name = billingAddress?.fullNameWithCompany
-        cell.address = billingAddress?.formattedPostalAddress ??
-            NSLocalizedString("No address specified.",
-                              comment: "Order details > customer info > billing details. This is where the address would normally display.")
-    }
-
-    func configureBillingEmail(cell: WooBasicTableViewCell) {
-        guard let email = order.billingAddress?.email else {
-            return
-        }
-
-        cell.bodyLabel?.text = email
-        cell.bodyLabel?.applyBodyStyle()
-        cell.accessoryImage = .mailImage
-
-        cell.isAccessibilityElement = true
-        cell.accessibilityTraits = .button
-        cell.accessibilityLabel = String.localizedStringWithFormat(
-            NSLocalizedString("Email: %@",
-                              comment: "Accessibility label that lets the user know the billing customer's email address"),
-            email
-        )
-
-        cell.accessibilityHint = NSLocalizedString(
-            "Composes a new email message to the billing customer.",
-            comment: "Accessibility hint, informing that a row can be tapped and an email composer view will appear."
-        )
+    func configureCustomerNote(cell: CustomerNoteTableViewCell) {
+        cell.headline = Title.customerNote
+        cell.body = "“\(customerNote)“"
+        cell.selectionStyle = .none
     }
 
-    func configureBillingPhone(cell: WooBasicTableViewCell) {
-        guard let phoneNumber = order.billingAddress?.phone else {
-            return
-        }
-
-        cell.bodyLabel?.text = phoneNumber
+    func configureBillingDetail(cell: WooBasicTableViewCell) {
+        cell.bodyLabel?.text = Footer.showBilling
         cell.bodyLabel?.applyBodyStyle()
-        cell.accessoryImage = .ellipsisImage
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
 
-        cell.isAccessibilityElement = true
         cell.accessibilityTraits = .button
-        cell.accessibilityLabel = String.localizedStringWithFormat(
-            NSLocalizedString(
-                "Phone number: %@",
-                comment: "Accessibility label that lets the user know the data is a phone number before speaking the phone number."
-            ),
-            phoneNumber
+        cell.accessibilityLabel = NSLocalizedString(
```

Comment:
I can not find the exact link to the accessibility guidelines, but as far as I recall the label should not include the type of control. There is a mention to it in the accessibility section of UIControl: https://developer.apple.com/documentation/uikit/uicontrol

Reply:
I used the Accessibility Inspector to check, and when I Run an Audit, I get some warning about this issue, not only in this specific case but all the accessibility labels used inside this class.
I'm going to fix them. 👍